src/main/java/net/snowflake/client/core/SFArrowResultSet.java

```diff
@@ -92,6 +95,8 @@ public class SFArrowResultSet extends SFBaseResultSet implements DataConversionC
    */
   private boolean formatDateWithTimezone;
 
+  protected Converters jsonConverters;
```

Comment:
it's protected in public class - should we mark it as internal?

Reply:
ok